lms/templates/courseware/courseware.html

```diff
@@ -186,8 +183,8 @@
         </div>
       % endif
 
-      <div id="accordion" style="display: none">
-        <nav aria-label="${_('Course Navigation')}">
+      <div class="accordion">
+        <nav class="course-navigation" aria-label="${_('Course Navigation')}">
```

Comment:
@clrux sorry I did not catch this the first time, but there is no need for the word "Navigation" in the aria-label since the screen reader already reports the role of the element, which is "navigation" so I am hearing "Course Navigation navigation"